test/README.rst

```diff
@@ -0,0 +1,64 @@
+Testing and debugging
+=====================
+Requirements
+------------
+Before running the test, install the dependencies::
+
+    pip install requirements.txt
+    pip install requirements-dev.txt
+
+
+Unit and Acceptance Tests
+-------------------------
+
+The test directory contains everything needed to run Selenium2Library
+tests with Robot Framework. This includes:
+
+- Unit tests under `unit` directory.
+- Acceptance tests written with Robot Framework under `acceptance`
+  directory
+- A very simple `httpserver.py` which is used to serve the html for tests in
+  `resources/testserver`
+- A collection of simple html files under `resources/html` directory
+- Start-up scripts for executing the tests: `run_tests.py` and
+  `run_unit_tests.py`
+
+To run unit and acceptance tests, run::
+
+    python test/run_tests.py ff|ie|chrome
+
+Tests are executed using Robot Framework and results verified automatically
+afterwards using `robotstatuschecker` tool. The tool can be installed using
+`pip install robotstatuschecker` and more information about it can be found
+from: https://github.com/robotframework/statuschecker/. Notice that initially
+some tests fail.
+
+
+It is possible to pass Robot Framework command line arguments to the test
+execution as a last arguments to the `run_tests.py` command. Example arguments
+like --test, --suite, --include and --exclude may be used to configure
+acceptance test execution.
```

Comment:
Do all RF's options work or can you only use some? Earlier versions had a restriction to use only some but is that fixed?